cmd/env/removerole.go

```diff
@@ -45,7 +45,7 @@ var RemoveRoleCmd = &cobra.Command{
 func init() {
 	RemoveRoleCmd.Flags().StringVarP(&memberName, "name", "n",
 		"", "Member name. Prefix the member role as role:name. Ex: user:foo@acme.com")
-	RemoveRoleCmd.Flags().StringVarP(&role, "role", "r",
+	RemoveRoleCmd.Flags().StringVarP(&role, "role", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `role` is breaking change right?